api/C/include/csi_csr_access.h

```diff
@@ -18,18 +18,13 @@
  */
 #define __STR(s)                #s
 #define STRINGIFY(s)            __STR(s)
+#define __ASM                   __asm
 
 /*
- * CSR operation Macro for csrrw instruction
- *
- * Read the content of csr register to __v,
- * then write content of val into csr register, then return __v.
- *
- * @param csr: CSR macro definition
- * @param val: value to store into the CSR register
- * @return : the CSR register value before written
+ * Read the content of csr register to __v, then write content of val into csr
```

Comment:
Missing argument, it might not be able to generate doxygen doc like this,

![image](https://github.com/riscv-non-isa/riscv-rvm-csi/assets/1538922/a7a21164-bc07-46cb-8549-53e9442b75a6)

Reply:
I can extend the schema to support a macro with documented arguments.  But let's treat that separately from this PR.